app/api/events.py

```diff
@@ -51,6 +51,18 @@
 from app.models.stripe_authorization import StripeAuthorization
 
 
+def check_ticketing(user, modules, data=None):
+    if user.is_verified:
+        raise ForbiddenException({'source': '/data/attributes/is-ticketing-enabled'},
+                                 "Please enter the location of the event")
```

Comment:
If user is not verified, throw that error and then check properly if location is not defined. What has user being verified got to do with location of the event?

Also all attribute keys are wrong, why are they 'data/attributes/is-ticketing-enabled'?